package.json

```diff
@@ -43,33 +43,12 @@
         "curl"
       ]
     },
-    "pacman": {
```

Comment:
Hmm, maybe is for the best, but would be good to listen to the community, maybe someone is actually downloding the pacman files. Even though I never did, always used AUR for that. 🤔